### ISSUE TITLE
server setup issue

### ISSUE THREAD

**Issue by aspenx** [DataViva/dataviva-site#368]
Hello,

I am trying the set up the server following the instruction given here. 
https://github.com/DataViva/dataviva-site

I have installed the site package on a VirtualBox VM(Ubuntu 14.04 guest on Windows 7 host). The installation process went smoothly. Due to hardware limitation, i have stripped large chunks of data off from some huge data files, such as "data_rais_ybio.sql", prior to importing into MySql. 

When executing the command ``python run.py runserver``, an error was generated (see end of message). It appears that the error is generated since the database table ``ei_yms`` is empty. 

I used ``https://github.com/DataViva/dataviva-scripts/blob/master/scripts/ei/ei_create_tables.sql`` to set up a bunch of tables in MySql. Somehow configuring these tables were not described in the site setup instruction. How are these table populated? Did I miss anything in order to set up the server?

Thanks
aspen

error message:

2015-12-30 16:43:25,051 INFO sqlalchemy.engine.base.Engine SELECT DATABASE()
2015-12-30 16:43:25,052 INFO sqlalchemy.engine.base.Engine ()
2015-12-30 16:43:25,108 INFO sqlalchemy.engine.base.Engine SHOW VARIABLES LIKE 'character_set%%'
2015-12-30 16:43:25,109 INFO sqlalchemy.engine.base.Engine ()
2015-12-30 16:43:25,109 INFO sqlalchemy.engine.base.Engine SHOW VARIABLES LIKE 'sql_mode'
2015-12-30 16:43:25,110 INFO sqlalchemy.engine.base.Engine ()
2015-12-30 16:43:25,110 INFO sqlalchemy.engine.base.Engine BEGIN (implicit)
2015-12-30 16:43:25,111 INFO sqlalchemy.engine.base.Engine SELECT ei_yms.year AS ei_yms_year 
FROM ei_yms ORDER BY ei_yms.year DESC 
 LIMIT %s
2015-12-30 16:43:25,112 INFO sqlalchemy.engine.base.Engine (1,)
Traceback (most recent call last):
  File "run.py", line 1, in <module>
    from dataviva import app
  File "/home/aspen/dv/dataviva/dataviva/__init__.py", line 65, in <module>
    __year_range__ = get_or_set_years(view_cache, "general:data_years")
  File "/home/aspen/dv/dataviva/dataviva/stats/util.py", line 73, in get_or_set_years
    val = compute_table_years(['ei', 'hedu', 'sc', 'secex', 'rais', 'population', 'stats'])
  File "/home/aspen/dv/dataviva/dataviva/stats/util.py", line 60, in compute_table_years
    max_year = str(get_year(tables[dataset], mode='max'))
  File "/home/aspen/dv/dataviva/dataviva/stats/util.py", line 38, in get_year
    return table.query.with_entities(table.year).order_by(year_col).first().year
AttributeError: 'NoneType' object has no attribute 'year'

**Comment by jspeis** [comment]
You might be using an older version of the code? If so can you try using the latest version? The ei references were removed in c081b21 back in October

**Comment by aspenx** [comment]
Thanks jspeis! The error disappeared after I retrieved the latest package. However I have a new problem.

After I run the python command, I got the following prompt in the terminal:

 * Running on http://127.0.0.1:5000/
 * Restarting with reloader

When using Firefox to access the local site, the following is displayed in the terminal with no other information:
127.0.0.1 - - [30/Dec/2015 21:49:42] "GET / HTTP/1.1" 302 -

So it appears the connection request is made. But Firefox shows "waiting for 127.0.0.1" in the status bar and after a minute or two, it displays "Server not found
Firefox can't find the server at en.127.0.0.1.".

I did add "127.0.0.1 en.localhost" to /etc/hosts.

What might be the problem? 

Thanks again.
